dockernet/dockerfiles/Dockerfile.relayer

```diff
@@ -21,5 +21,5 @@ USER 1000
 WORKDIR /home/relayer
 
 # Create startup script that takes a path as a parameter
-RUN echo 'rly start $1 -p events' > start.sh
+RUN echo 'rly start $1' > start.sh
```

Comment:
😮 

we can get rid of "-p events" now?!

Reply:
yessir!